Release MongooseIM 6.2.1 (#4254)
esl/MongooseIM: CHANGELOG.md

# [MongooseIM 6.2.1](https://github.com/esl/MongooseIM/releases/tag/6.2.0) - 2024-04-09

## Highlights

- Enhanced CETS
- Pools configuration
- Traffic shapers update
- Support for MAM v1.1.0

> - Added Erlang Doctor

Reviewer: I'd remove this from highlights, because it's a low-level debugging tool.

- More reliable testing
- Various improvements and fixes

## All changes

### Added
- RDBMS backend for `mod_caps` (#4211)
- Commit hash to GraphQL server status (#4221)

> - Erlang Doctor for diagnostics (#4248)

Reviewer: ```suggestion
- Erlang Doctor debugging tool (#4248)
```

- Pools:
  - By host type (#4229)
  - By host config (#4235)
  - Names to `mongoose_rdbms` (#4231)

### Changed
- Improved shapers (#4187, #4203, #4213)
- Enhanced discovery requests handling (#4194)

> - User-friendly errors for internal database issues (#4192)

Reviewer: ```suggestion
- User-friendly errors for internal databases in the GraphQL API (#4192)
```

- Increase `idle_timeout` for SSE (#4196)
- Presence management refactor (#4207)

> - MAM implementation update (#4218, #4225)

Reviewer: ```suggestion
- MAM implementation update to version 1.1.0(#4218, #4225)
```

- Roster management refactor (#4209)

### Fixed
- CETS improvements:
  - Pause on all nodes (#4204)
  - Node cleanup (#4234, #4250, #4251)
  - Unnecessary logs removal (#4205)

> - MAM lookup (#4191)

Reviewer: ```suggestion
- MAM lookup error handling (#4191)
```

> - MUC light `id` definition for MariaDB (#4195)

Reviewer: ```suggestion
- MUC Light `id` definition for MariaDB (#4195)
```

> # [MongooseIM 6.2.1](https://github.com/esl/MongooseIM/releases/tag/6.2.0) - 2024-04-10

Reviewer: ```suggestion
# [MongooseIM 6.2.1](https://github.com/esl/MongooseIM/releases/tag/6.2.1) - 2024-04-10
```
